dataactbroker/handlers/fileHandler.py

```diff
@@ -195,8 +195,8 @@ def checkSubmissionPermission(self,submission):
         """
         userId = LoginSession.getName(session)
         user = self.interfaces.userDb.getUserByUID(userId)
-        # Check that user has permission to see this submission, user must either own the submission or be an admin
-        if(submission.user_id != userId and not self.interfaces.userDb.hasPermission(user,"website_admin")):
+        # Check that user has permission to see this submission, user must be within the agency of the submission
+        if(submission.cgac_code != user.cgac_code):
```

Comment:
Can a user do anything to submissions within their own agency or just view them?  This change allows users to update these submissions, and eventually "delete" or submit them to data store.